rfs/nostr-blossom/src/client.rs

```diff
@@ -59,7 +59,7 @@ impl BlossomClient {
     where
         T: NostrSigner,
     {
-        let url = format!("{}/upload", self.base_url);
```

Comment:
Thanks! What do you think if we use the `join` method instead?

```rust
let url: Url = self.base_url.join("upload")?;
```

The same in the other parts in this module